invenio_app_ils/circulation/search.py

```diff
@@ -65,6 +65,15 @@ def get_active_loan_by_item_pid(self, item_pid):
             ),
         )
 
+    def get_overdue_loans_by_doc_pid(self, document_pid):
+        """Return any overdue loans for the given document."""
+        return search_by_pid(
+            document_pid=document_pid,
+            filter_states=current_app.config.get(
+                "CIRCULATION_STATES_LOAN_ACTIVE", []
+            ),
+        ).filter('range', request_expire_date=dict(lt='now/d'))
```

Comment:
I think these are overdue `requests`, not `loans`, or am I wrong? Shouldn't be more the `end_date` for a loan?